api/indexer.oas3.yml

```diff
@@ -1041,7 +1041,7 @@
             "properties": {},
             "type": "object"
           },
-          "db-unavailable": {
+          "db-available": {
```

Comment:
was `db-unavailable` released? Will anyone be sad if it goes away?

Reply:
db-UNavailable has the advantage that it can be default-false and not present in the json in the default state. It can be `{'db-unavailable':true}` or just not there at all.

Reply:
It hasn't been released yet, so this is just in time.

A lot of the public fields (including this one) is marked to serialize regardless of whether or not it's a zero value, but that's an interesting point that I usually don't think about.